bindata/manifests/daemon/daemonset.yaml

```diff
@@ -0,0 +1,49 @@
+
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ingress-node-firewall-daemon
+spec:
+  selector:
+    matchLabels:
+      app: ingress-node-firewall-daemon
+  template:
+    metadata:
+      labels:
+        app: ingress-node-firewall-daemon
+        component: network
+        type: infra
+    spec:
+      containers:
+        - command: ['/bin/sh', '-c', 'mount bpffs /sys/fs/bpf -t bpf && daemon']
+          image: '{{.Image}}'
+          name: daemon
+          securityContext:
+            privileged: true
+            runAsUser: 0
+            capabilities:
+              add:
+                - CAP_BPF
+                - CAP_NET_ADMIN
+          volumeMounts:
+            - name: bpf-maps
+              mountPath: /sys/fs/bpf
+              mountPropagation: Bidirectional
+      nodeSelector:
+        kubernetes.io/os: linux
+      serviceAccountName: daemon
+      shareProcessNamespace: true
+      terminationGracePeriodSeconds: 2
+      tolerations:
```

Comment:
Remove that toleration here, you're setting it programmatically via the configuration crd